tf-modules/archive/fetch_or_create_rsa_keys.sh

```diff
@@ -2,10 +2,6 @@
 
 set -e
 
-apt-get update
-apt-get install -y python-pip
-pip install awscli
```

Comment:
The script doesn't need to ensure that the AWS CLI exists?

Reply:
It definitely needs to exist. The problem is that the `apt-get` commands were assuming that the script would only ever be run on a Debian-like system. If we deploy from our laptops, though, that apt-get command won't exist. When we write documentation, we're going to just list "must have aws-cli installed" as a pre-req.

Reply:
This was specifically a hack to make our CI deploy work. We can reintroduce it better when we update the CI build, or prioritize CUMULUS-1495.